### PR TITLE
Flights Route

### PR DESCRIPTION
**Pull request by nmking22**
This PR will:
* Create fully functioning /flights path to handle requests for use in both flight search and lucky flight functionality
* Refactor with poro and serializer
* Add default parameters for calls
* Test sad paths, edge cases, and all default params

- [x] Repo Representation: Flight-API

- [x] Sad Path Tested

- [x] Happy Path Tested

- [x] All Tests are passing

- [x] Updated README.md with added information

- [x] Collaborators: Jesse Mellinger

### CODE REVIEW

File: app/poros/flight.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @origin_city = flight_data[:cityFrom]
+    @destination_city = flight_data[:cityTo]
+    @origin_iata = flight_data[:flyFrom]
+    @destination_iata = flight_data[:flyTo]
+    @latitude = flight_data[:route][0][:latTo]
+    @longitude = flight_data[:route][0][:lngTo]
+    @price = flight_data[:price]
+    @departure_datetime = DateTime.strptime(flight_data[:dTime].to_s,'%s')
+    @arrival_datetime = DateTime.strptime(flight_data[:aTime].to_s,'%s')
+    @r_departure_datetime = DateTime.strptime(flight_data[:route][1][:dTime].to_s,'%s')
```
Comment (stillsheryl) [inline app/poros/flight.rb:30]: Nice job figuring out the time conversion!

Comment (dunlapww) [reply app/poros/flight.rb:30]: Great work!  Do we know which time zone is being reported?  We prob need a user story at some point to adjust timezone to local timezone - maybe timezone is converted in the view?

**Review by dunlapww — APPROVED** [review]
Great work guys!  Nice job incorporating fastjson.  That may make it much easier if we decide we want more or different attributes at a later point.

**Review by GeorgieGirl24 — APPROVED** [review]

File: app/poros/flight.rb (hunk 1 above)
Comment (nmking22) [reply app/poros/flight.rb:30]: Jesse and I were thinking that the local time zone is always being reported. If you look at this [link](https://www.kiwi.com/en/booking?activeStep=0&affilid=picky&currency=usd&deeplinkId=28432034945&flightsId=0713244648e10000956fd20d_0-2446071348e60000bdad426d_0&handBags=null&holdBags=null&lang=en&passengers=1&price=39.26&session_identifier=YbBk9Zoa8kzQyPJPaEDvG9ds5%2Bazz9t8vsNCU5jE5sg%3D&session_token=rqJ4%2FTwVa2pvkvf6CyJ216z0r2kf8zjsyBXDIcNXXnxFxQEmHmI7ciSf86s4V3vlGWfyDuNIAwgkvtIOVoBQMC06kOlnE0C6PxJMh2ti50vb5ZMrrc98kOYYERUsNPLZCIP0bPlqiK4aLxZPDjEawTlD3gxAT23m9YlsolT5jUElJxCXQIprEwM1GBew2NoNmJqV87J7vr%2B7zylL8hRsHwlIlhVTZmnJle7%2B1b8k6Yc%3D&token=B5XrXDUgnGxN7_jWzhQseOjAvcAuXwfmyhFZZcoJ6D3RKTk8AJX2zNufzwiJmvlBCk7ez16n6r8arrAz9BfnASFGkoMGOF9RWVlgbl4dmvrjQJ53CFuqqqoZklpI4qc1qD5davs6ATzkPmujtP99ShKXtaROZksJiL6lq9spZY88A00df85ixoqejtd06UEHK_dr6Vyi6T3eSlWRCZWfJrZpdMLPmfTsRZRm-9gXL380rdNfC3aowszk3-0eUIb5qu0LE0BWCA-q7V6J0zihohDTPN6QH6rxXBSos1e6o_QVl5VkKouSjEHXvfXqkwbVWwEdPtOi2BJIVakYKPDZyKE25g7auPWdNxUKZJ7WqRZmiepisTdlsWQIU9cBaNTClZNzRa0MpehM2rUynvQfN-nTXtmzQGZC1wVwIj1ErelaSyeJMCLkk41Aa-SCityd7ERbrhj5GK6F8SnKKxf7wCu1qPaYRlAxBKfmgpEVki8TgX8z7vnIWx8oA2cI4tf4hEf7v0vo49pCGHSZN6jJlY-_4eC74jazxHT_vLsRzNG5H52pbiSh9DaJjoOZOix9OshAHhYeE9XsfuVVavkNrv5C2cLGT0FihbPo4T_20jCilsSN5DBq_YDZH08zQw3UV-oUPaYD9GlUqTS_dSvlOnBVcOJuSY5Y4abfBXJLf-cM%3D&user_id=4eaef74e-adbb-4baa-9d9d-672b2abf9f5e), both flights have flight times of around 2 hours, but the expected time differences in the flights are 1 hour and 3 hours implying that the time change from MTN to PST is being accounted for. We don't have a good idea of how we could explicitly display the time zones in our views though if that's something we'd like to present to the user.
